store/model/alerts.go

```diff
@@ -1,14 +1,13 @@
-package alerts
+package model
```

Comment:
can get rid this `model` package and have the model definitions inside `store/postgres` package all together, since the model definitions are specific to postgres and/or gorm

Reply:
fyi we're going to do the same for guardian